src/ispc/kernels/rescale_alpha.ispc

```diff
@@ -0,0 +1,120 @@
+#include "image.ispc"
+
+float get_alpha(uniform Image image, uint64 x, uint64 y) {
+    uint64 pixel_index = x + y * image.size.x;
+    return (float)image.data[pixel_index * 4 + 3] / 255.0f;
```

Comment:
Pretty sure we support 3- and 4-channel images.

Reply:
There's an assert prior to calling this to only do this on 4-channel textures from rust. It doesn't make sense calling this on a 3-channel texture

Reply:
Then at least make it a constant so that this math is a bit more clear.